velox/docs/monthly-updates/december-2023.rst

```diff
@@ -0,0 +1,99 @@
+********************
+December 2023 Update
+********************
+
+Documentation
+=============
+
+* Add documentation for :doc:`Runtime Metric</metrics>`.
```

Comment:
typo? Runtime Metric -> .... Metrics

Reply:
I felt it was incorrect too, but the document mentions `Runtime Metric`. 
https://facebookincubator.github.io/velox/metrics.html
Should we improve that doc?

Reply:
Yes, looks like this doc has some typos. CC: @xiaoxmeng

Reply:
I made it `Runtime Metrics` here. Let's fix the doc in a separate PR.